configure.ac

```diff
@@ -131,6 +131,8 @@ AC_SUBST(AR_R)
 
 AC_PATH_PROG(LINUXDOC, linuxdoc, $FALSE)
 AM_CONDITIONAL(ENABLE_RELEASE_DOCS, test "x${ac_cv_path_LINUXDOC}" != "x$FALSE")
+SQUID_RELEASE=`echo $VERSION | cut -d. -f1`
+AC_SUBST(SQUID_RELEASE)
```

Comment:
Since `SQUID_RELEASE` is generally-useful information that is provided even when `ENABLE_RELEASE_DOCS` is falsy, please separate these two new lines from `ENABLE_RELEASE_DOCS` code above them. I also recommend moving them higher, to make `SQUID_RELEASE` available to a larger section of future `configure.ac` code.

Reply:
What future autoconf code do you envision needing the Squid series number earlier than this?
The lines above here are all detecting the build environment/toolchain. This is the point in configure.ac where it begins determining the values going into output files.

I could document those areas better if you like.

Reply:
> What future autoconf code do you envision needing the Squid series number earlier than this?

To save time, I will not try to defend my suggestion to make `SQUID_RELEASE` available to a larger section of future `configure.ac` code. Ignore it if you think it is wrong.

Still, please separate these two new lines from `ENABLE_RELEASE_DOCS` code above them (for the reasons already given).

> I could document those areas better if you like.

I am not requesting any area documentation improvements in this change request, and I do not recommend making them in this PR.

Reply:
I support this going in. It doesn't hurt, and it can help